Tasks/AzureFileCopyV2/AzureUtilityGTE1.0.ps1

```diff
@@ -24,7 +24,7 @@ function Get-AzureStorageAccountResourceGroupName
     if (-not [string]::IsNullOrEmpty($storageAccountName))
     {
         Write-Verbose "[Azure Call]Getting resource details for azure storage account resource: $storageAccountName with resource type: $ARMStorageAccountResourceType"
-        $azureStorageAccountResourceDetails = Get-AzureRmResource -ResourceType $ARMStorageAccountResourceType -Name $storageAccountName -ErrorAction Stop
+        $azureStorageAccountResourceDetails = Get-AzureRmResource -ErrorAction Stop | Where-Object { ($_.ResourceType -eq $ARMStorageAccountResourceType) -and ($_.Name -eq $storageAccountName)}
```

Comment:
Just remember now, AzureFilecopyV4 also uses azureRm module, we have not done that change in master branch na?
Master branch still uses AzureRmStorageAccount commandlet in AzureFileCopyV4

Reply:
yes I know. I will first validate the changes in V2 and V3 and then do it in V4.